src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -0,0 +1,1364 @@
+                }
+
+                case ContrastSecurityRuleIds.FormsAuthenticationSSL:
+                {
+                    return ConstructFormsAuthenticationSSLResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.FormsWithoutAutocompletePrevention:
+                {
+                    return ConstructFormsWithoutAutocompletePreventionResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.HttpOnlyCookieFlagDisabled:
+                {
+                    return ConstructHttpOnlyCookieFlagDisabledResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.InsecureEncryptionAlgorithms:
+                {
+                    return ConstructInsecureEncryptionAlgorithmsResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.OverlyLongSessionTimeout:
+                {
+                    return ConstructOverlyLongSessionTimeoutResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.PagesWithoutAntiClickjackingControls:
+                {
+                    return ConstructPagesWithoutAntiClickjackingControlsResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.PathTraversal:
+                {
+                    return ConstructPathTraversalResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.RequestValidationModeDisabled:
+                {
+                    return ConstructRequestValidationModeDisabledResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.SessionCookieHasNoSecureFlag:
+                {
+                    return ConstructSessionCookieHasNoSecureFlagResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.SessionRewriting:
+                {
+                    return ConstructSessionRewritingResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.SqlInjection:
+                {
+                    return ConstructSqlInjectionResult(context);
+                }
+
+                case ContrastSecurityRuleIds.VersionHeaderEnabled:
+                {
+                    return ConstructVersionHeaderEnabledResult(context.Properties);
+                }
+
+                case ContrastSecurityRuleIds.WebApplicationDeployedinDebugMode:
+                {
+                    return ConstructWebApplicationDeployedinDebugModeResult(context.Properties);
+                }
+
+                default:
+                {
+                    return ConstructNotImplementedRuleResult(context.RuleId);
+                }
+            }
+
+            throw new InvalidOperationException();
+        }
+
+        private Result ConstructNotImplementedRuleResult(string ruleId)
```

Comment:
>private [](start = 8, length = 7)

And now I have to confess that I didn't review the remaining 1000 lines. I suppose I really should, huh?

What happened was, I reviewed enough of it to understand what I had to do to make it work with 2019-01-09, and then I stopped because I thought it important to get this in to `master`. How about this -- we merge this, to give you and/or me or Harleen the ability to move forward, and I review the existing code _post hoc_, and submit a PR with my suggested changes.